exotations/exotica_core_task_maps/src/look_at.cpp

```diff
@@ -36,30 +36,15 @@ namespace exotica
 LookAt::LookAt() = default;
 LookAt::~LookAt() = default;
 
-Eigen::Vector3d LookAt::get_look_at_target_in_world(const int& i)
-{
-    if (i >= n_end_effs_ || i < 0) ThrowPretty("Out of bounds, got " << i << " but expected less than " << n_end_effs_);
-    return Eigen::Map<Eigen::Vector3d>(kinematics[0].Phi(n_end_effs_ * i + 2).p.data);
-}
-
 void LookAt::Update(Eigen::VectorXdRefConst x, Eigen::VectorXdRef phi)
 {
     if (phi.rows() != TaskSpaceDim()) ThrowNamed("Wrong size of phi!");
 
     for (int i = 0; i < n_end_effs_; ++i)
     {
-        const int end_effector_id = i * n_end_effs_;
-
-        // Get EffPoint and LookAtTarget
-        Eigen::Vector3d c = Eigen::Map<Eigen::Vector3d>(kinematics[0].Phi(end_effector_id).p.data);      // EffPoint | Eff frame
-        Eigen::Vector3d p = Eigen::Map<Eigen::Vector3d>(kinematics[0].Phi(end_effector_id + 1).p.data);  // LookAtTarget | Eff frame
-
-        // Compute orthogonal orthogonal projection a onto line e->c
-        double alpha = p.dot(c) / c.squaredNorm();
-        Eigen::Vector3d a = alpha * c;
-
-        // Set Phi
-        phi.segment<3>(end_effector_id) = a - p;
+        Eigen::Vector3d p = Eigen::Map<Eigen::Vector3d>(kinematics[0].Phi(i).p.data);
```

Comment:
From this, it tells me that now the equation simplifies to be the same same as `EffPosition`, but only considering x and y. Is this correct?